charts/argo-cd/Chart.yaml

```diff
@@ -25,4 +25,4 @@ dependencies:
   - name: argocd-image-updater
     version: 0.1.0
     repository: https://charts.akuity.io
-    condition: imageUpdater.enabled
+    condition: argocd-image-updater.enabled, imageUpdater.enabled
```

Comment:
Remove `imageUpdater.enabled`?

Reply:
I was thinking if somebody was using an older chart they may still have that value in place to disable it, so that is for backward compatibility. While the rest of the values weren't being passed down, imageUpdater was still and should still work to disable the subchart.